src/BookReaderComponent/BookReaderComponent.js

```diff
@@ -4,59 +4,65 @@
 
 import { LitElement, html, css } from 'lit-element';
 
-import '../ItemNavigator/ItemNavigator.js';
-import '../BookNavigator/BookNavigator.js';
-
+import '@internetarchive/ia-item-navigator';
+import '../BookNavigator/book-navigator.js';
+import '@internetarchive/modal-manager';
+import { SharedResizeObserver } from '@internetarchive/shared-resize-observer';
 export class BookReader extends LitElement {
   static get properties() {
     return {
-      base64Json: { type: String },
+      item: { type: String },
```

Comment:
Q: Is `item` the metadata? If so, should it be `type: Object`?

Reply:
i remember adding this value because `BookReaderJSIA` needs to be base64 string from the metadata value which is on `type: Object`

Reply:
thanks. base64 was artifact from SSR load, but we will change this on PB side. this will normalize it. updated type